src/coprocessor/dag/builder.rs

```diff
@@ -60,6 +60,19 @@ impl DAGBuilder {
                         }
                     }
                 }
+                ExecType::TypeIndexScan => {
+                    let descriptor = ed.get_idx_scan();
+                    for column in descriptor.get_columns() {
```

Comment:
How about moving this logic outside `match`?

Reply:
this descriptor is not that descriptor. They are different types.

Reply:
got it.